webapp/components/Form.tsx

```
@@ -123,9 +108,30 @@ const Response = ({
+  const [resetForm, setResetForm] = useState<boolean>(false);
   const [success, setSuccess] = useState<boolean | null>(null);
   const [response, setResponse] = useState<string | null>(null);
 
+  const initialValues = useMemo<FormValues>(
+    () => ({
+      name: getFromLocalStorage('name'),
+      mailFrom: getFromLocalStorage('mailFrom'),
+      group: getFromSessionStorage('group'),
+      mailTo: getFromSessionStorage('mailTo'),
+      accountNumber: getFromSessionStorage('accountNumber'),
+      amount: isNaN(parseInt(getFromSessionStorage('amount'), 10))
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)